docs/iqe-ee-proxy.md

```diff
@@ -0,0 +1,44 @@
+# Setup for the IQE to Ephemeral Proxy
+
+### This allows a user to run and debug the IQE python tests locally against a deployment on OpenShift
+
+1. Create an iqe_workspace directory  
+2. Copy get-iqe.py to directory:  
```

Comment:
Also, update the .gitignore to exclude this directory "iqe_workspace"

Reply:
It does not need to be in rhsm-subscriptions. I run it outside of it.